.github/workflows/tag_image_push_rhel.yml

```diff
@@ -34,6 +34,9 @@ jobs:
       PROJECT_NAME: hztest-${{ github.run_id }}-${{ github.run_attempt }}
 
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        jdk: [ '11', '17', '21' ]
```

Comment:
use jdk variable in ` Get OpenShift events` and `Store OpenShift events as artifact`

Reply:
done